R/extractFDAFeaturesMethods.R

```diff
@@ -106,77 +126,295 @@ extractFDAFourier = function(trafo.coeff = "phase") {
 #' @export
 #' @family fda_featextractor
 extractFDAWavelets = function(filter = "la8", boundary = "periodic") {
-  assertCharacter(filter)
-  assertChoice(boundary, c("periodic", "reflection"))
 
-  lrn = function(data, target = NULL, col, filter, boundary) {
+  # All possible values for the filters
+  filter.vals = c(
+    paste0("d", c(2, 4, 6, 8, 10, 12, 14, 16, 18, 20)),
+    paste0("la", c(8, 10, 12, 14, 16, 18, 20)),
+    paste0("bl", c(14, 18, 20)),
+    paste0("c", c(6, 12, 18, 24, 30)),
+    "haar"
+  )
 
```

Comment:
This is valid. The parameters are a combination of characters (i.e. `d`) and the wavelet length (i.e. `2`). 
Instead of writing all of them down I decided to be lazy here.